### PR TITLE
Lavaland Clock Cult Ruin

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
dead

RIP clockwork cult

WJ did the mapping, spessmenart did the dead ratvar, I just threw it all together

### CODE REVIEW

**Comment by Shadowflame909** [comment]
Is this ironic compared to how dead clock cult is now? I miss it...

File: _maps/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm [hunk 1]
```diff
@@ -0,0 +1,60 @@
+"a" = (/turf/template_noop,/area/template_noop)
```
Comment (81Denton) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm]: Looks like mapmerge didn't work

**Comment by msgerbs** [comment]
Correct me if I'm wrong but brass spears don't work for non-cultists anyway right?

File: _maps/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm (hunk 1 above)
Comment (KorPhaeron) [reply _maps/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm]: Looks like I forgot to run it*

**Comment by 81Denton** [comment]
The sprite and lighting work together really well. Oof, this hurts my heart.

**Comment by iskyp** [comment]
i feel sad for a clock now ty

**Comment by PatchouliKnowledge** [comment]
RIP clucks.
Maybe throw in a replica fabricator instead of that spear? Feels like that could be more fun.

**Comment by Repukan** [comment]
Will this stay in after (if) CC gets removed?

**Comment by KorPhaeron** [comment]
@Repukan 

Yeah. I think "Nanotrasen blew up the city of cogs and debris fell into lavaland and now there isn't any more clock cult" is a more fun explanation than "we removed the game mode with no trace"

@PatchouliKnowledge Is there a version that isn't cult only?

**Comment by PKPenguin321** [comment]
Nanotrasen is canonically a dimensionkiller, nice

**Comment by praisenarsie** [comment]
tbh i wish there was better lighting so you can see the carcass of ratvar better

**Comment by Ratvar** [comment]
I am not sure rock splinters mesh well together with clockwork floor under

**Comment by Wilchenx** [comment]
Something something wasting ruin slot

**Comment by Ratvar** [comment]
Something something ruin that's not loot piniatta

**Comment by Bluespace** [comment]
throw in my clockwork titan megafauna here

**Comment by GuyonBroadway** [comment]
Clock cult is such a great theme its a shame it never got worked into a decent game mode.

Ah well, we shoukd keep the assets and shit cus they cool.

**Comment by GuyonBroadway** [comment]
Actually gives me a funny idea, what if you could make the red eye flicker ever so slightly, even dead gods dream after all.

**Comment by Armhulen** [comment]
@Wilchenx ruin costs zero YOU GOD DAMNED DOOFUS.

**Comment by Therapist1** [comment]
Well, guess I just got to accept the death, this makes it a lot easier.

**Comment by Wilchenx** [comment]
@Armhulen yeah but it takes place of another ruin

**Comment by WJohn** [comment]
It literally doesn't take the place of another ruin by definition.

**Comment by Bluespace** [comment]
why not color the rocks with bronze flecks.

**Review by ChangelingRain — APPROVED** [review]

**Comment by WJohn** [comment]
Oh, important thing to note, kor: You may want to place some nolava markers around the ruin to prevent lava rivers from running straight through this thing.

**Comment by Y0SH1M4S73R** [comment]
@PKPenguin321 Hold on a sec...... I thought clock fluff said that the proto-Neovgre that Nezbere made was the only thing other than Narsie that was capable of killing Ratvar. Did Nanotrasen somehow invent a Neovgre-tier weapon with their own technology, or does Ratvar merely lie dormant and crippled, awaiting the day that his servants can restore him........ IMHO, this provides a jumping-off point to brainstorm some possibly workable clock cult reworks.

**Comment by Repukan** [comment]
Doesn't matter because A) Official lore was never made and modt importantly

B) Nobody wants to rework this piece of shit

**Comment by lordsaladin** [comment]
Can we get clock cult pylons?

**Comment by octareenroon91** [comment]
Brass toolbox maybe?

**Comment by zaracka** [comment]
Whether Clockwork Cult comes back or not, I like the ruin and think it should exist. You could probably add a check to prevent it from spawning if the game mode is Clockwork. If I'm right about how ruins are generated, people can't use this to meta the game mode.

**Comment by octareenroon91** [comment]
>Whether Clockwork Cult comes back or not, I like the ruin and think it should exist. You could probably add a check to prevent it from spawning if the game mode is Clockwork. If I'm right about how ruins are generated, people can't use this to meta the game mode.

Gamemode is set after the map loads and gets ruins. Any fix would have to come through clockcult gamemode code. Just saying.

**Comment by zaracka** [comment]
Right... Guess there's no point in brainstorming at the moment.

**Comment by KorPhaeron** [comment]
@WJohn Ratvar is lava proof so it isn't the end of the world if he "lands" in a river

@davethwave There is a stack of brass and a shitty spear, not much to pillage

**Comment by Ratvar** [comment]
Make Ratvar deconstructable? Random clock parts, more brass, cables, shards, crystals, glass, iron rods infinite for as long as you deconstruct It?

**Comment by davethwave** [comment]
That doesn't feel right. How about staying too long around the ruin turns you into a clockwork golem that hears voices in clockcult. Kind of like a corruption effect.

**Comment by davethwave** [comment]
Also I think I would be neat if you click the big red thing at ratvar's chest with a SM shard it would turn you into a antag kind of like clockcult but with weaker spells and can't teleport to rebee since its "destroyed". The goal of said antag would be too feed Ratvar enough power to start fresh and retreat into the void to fully recover.(might work great as a rare event/admin spawn antag too)

**Comment by fludd12** [comment]
*adds remaking clock cult to the list of impossible things on my list to do*

**Comment by GuyonBroadway** [comment]
Imagine if that was clock cult but a side antag like old abuctors.

No converts, no round ending, no mass murder machines, the cultists just get a scavenger hunt list they need to bring to ratvars corpse to revive him.

Once you got everything, that noise that used to play when the clock cult win plays locally and ratvars corse "warps away" with some little thank you message written in ratvarian and the cultists get to be clockwork war golems. They greentext when the shuttle docks.

**Comment by iskyp** [comment]
>kor keep adding things to this gravestone

clock cult is dead stop requesting things for this ruin to do

**Comment by Repukan** [comment]
Those are all wonderful ideas. Tell me when you guys code them so we can bask in their cleverness.

**Pull request merged**